packages/api/package.json

```diff
@@ -9,15 +9,16 @@
     "deepclean": "npm run clean && rimraf node_modules",
     "dev": "NODE_EXTRA_CA_CERTS=opensearch-root-ca.pem ts-node-dev --transpile-only --respawn --inspect=0.0.0.0:9229 --project tsconfig.dev.json --files src/app.ts",
     "start": "node dist/app.js",
-    "start-opensearch": "docker compose -f docker-compose-opensearch.yml -p opensearch up -d && docker cp opensearch-node1:/usr/share/opensearch/config/root-ca.pem ./opensearch-root-ca.pem",
-    "start-docker-compose": "docker compose -f docker-compose.dev.yml up --build -d && npm run start-opensearch",
-    "start-dependencies": "docker compose -f docker-compose.dev.yml up postgres dynamodb -d && npm run start-opensearch",
+    "start-docker-compose": "docker compose -f docker-compose.dev.yml up --build -d && npm run start-opensearch && npm run start-analytics-transform",
+    "start-opensearch": "docker compose -f docker-compose-opensearch.yml -p opensearch up -d && docker cp ./opensearch-root-ca.pem opensearch-node1:/usr/share/opensearch/config/root-ca.pem",
+    "start-analytics-transform": "cd ../data-transformation/fhir-to-csv/ && docker compose -f docker-compose.server.yml up --build -d && cd -",
+    "start-dependencies": "docker compose -f docker-compose.dev.yml up postgres dynamodb -d && npm run start-opensearch && npm run start-analytics-transform",
```

Comment:
Modifications, in addition to reordering the script in the file:

<img width="1478" height="85" alt="image" src="https://github.com/user-attachments/assets/fd871287-2b41-43d6-af61-1d1acd7d32c2" />